primer/brand
# Card a11y fixes

## Summary

Addresses @TylerJDev's feedback in https://github.com/github/primer/issues/1809

Specifically:
- Hide native SVGs by default, similarly to the Pillar implementation
- Hide trailing SVG after the CTA link (ExpandableArrow)
- Adjust markup in documentation examples to show Card.Heading before Card.Label
- Ensure link text only contains text from Card.Heading, possibly by using pseudo-element pattern

## List of notable changes:
- The card container element is no longer `<a>`, it's `<div>`. **This might be a breaking change** because a `div` has different attributes than an `a`.

## What should reviewers focus on?
- Confirm that the feedback has been addressed
- Confirm there have been no visual regressions

## Steps to test:

1. Open the component in CI-deployed preview environment
2. Go to any of the Card stories in Storybook
3. Verify that the a11y eng feedback has been addressed
  - Hide native SVGs by default, similarly to the Pillar implementation
  - Hide trailing SVG after the CTA link (ExpandableArrow)
  - Adjust markup in documentation examples to show Card.Heading before Card.Label
  - Ensure link text only contains text from Card.Heading, possibly by using pseudo-element pattern
5. Verify that there are no visual or a11y regressions

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

N/A - there should be no visual changes
